coding/cpp_style_guide.rst

```diff
@@ -1984,432 +1982,64 @@ We should, of course, strive to minimize the amount of such information.
 
 .. _style-guide-cpp-6-1:
 
-6-1. Multiple statements per line SHOULD NOT be used.
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-.. code-block:: cpp
+6-1. We follow the clang-format Google style for code layout with a few exceptions.
```

Comment:
@PaulPrice if you mean an EUPS package with `clang-format` in it then I'm not sure why we need that (it's in homebrew for example). Are you asking for a `scons` target or something that can make it easy for people to update their code?